### DISCUSSION TITLE
Unvisible items on right panel

### DISCUSSION THREAD

**Discussion by hreba** [osmandapp/OsmAnd#17299] [General]
Hi,

First I want to thank everybody for that great app, really helpful.

My problem: I see only one item on the right panel as in this screenshot:
![shot1](https://github.com/osmandapp/OsmAnd/assets/76042774/95dde5e9-1d24-4c71-9e86-eae33a825025).
However, I configured 4 items, as on this screenshot:
![shot2](https://github.com/osmandapp/OsmAnd/assets/76042774/81671644-7836-4d30-a735-6f0ca3e5c68b)
Why are 3 of them unvisible?

Thanks in advance.

**Comment by mikehgentry** [comment]
That is weird... What happens if you start a navigation?

**Comment by hreba** [comment]
When I used Osmand for navigation another time, everything was displayed as it should. Now, sitting at home, just starting a navigation, but with navigation inactive due to missing location data, the display is as described in my first post.

Thinking well, it makes sense not to display unknown data: none of them is known as long as my position is unknown. Distance to designation then should be hidden too, though. Nevertheless, I am now content with the behaviour.

Thanks for the reply.
